tests/test_tag_matching.py

```diff
@@ -0,0 +1,49 @@
+# Temporary mock setup while waiting for a better way to do integration tests
+from unittest.mock import patch
+
+import pytest
+
+
+def mocked_post_request():
+    return {'cucoo': 'haha'}
+
+
+@patch('requests.post')
+@pytest.fixture(scope='module')
+def tagmatching_result(mock_post):
+    from cognite.tagmatching import tag_matching
+    mock_post.return_value.status_code = 200
```

Comment:
Only mocking status code and return value is risky if the functions try to do something more fancy than only reading those values. Try to use MockReturnValue in conftest.